### ISSUE TITLE
About data preprocessing

### ISSUE THREAD

**Issue by HWSQHL** [RUB-SysSec/GANDCTAnalysis#21]
In Section 4.1 of the paper，
“For training a model on samples in the image domain, we normalize the pixel values to the range [−1, 1]. In the frequency domain, we first convert the images using DCT, then log-scale the coefficients and, finally, normalize them by removing the mean and scaling to unit variance.”  
How do you process images? Can you give me a code to process  it.I want to use pytorch to realize it.

**Comment by v0lta** [comment]
Hi @HWSQHL, the code is already here. I think you are looking for:

https://github.com/RUB-SysSec/GANDCTAnalysis/blob/988c4c3dc46ada117f2bb49a03ebf57c292dc505/src/image_np.py#L39

and,

https://github.com/RUB-SysSec/GANDCTAnalysis/blob/988c4c3dc46ada117f2bb49a03ebf57c292dc505/prepare_dataset.py#L225
